tools/junit-openfeature/src/test/java/dev/openfeature/contrib/tools/junitopenfeature/DoubleFlagTest.java

```diff
@@ -29,6 +37,13 @@ void existingSimpleFlagIsRetrieved() {
             assertThat(client.getDoubleValue(FLAG, FALLBACK)).isEqualTo(FLAG_VALUE);
         }
 
+        @Test
+        @DoubleFlag(name = FLAG, value = 1.d)
```

Comment:
```suggestion
@DoubleFlag(name = FLAG, value = FLAG_VALUE)
```
also for the other tests